packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -1,46 +1,35 @@
-import { waitForPortOpen } from '@celo/dev-utils/lib/network'
+/* tslint:disable: no-console */
 import BigNumber from 'bignumber.js'
 import { assert } from 'chai'
-import { spawn, SpawnOptions } from 'child_process'
 import fs from 'fs'
 import _ from 'lodash'
 import { join as joinPath, resolve as resolvePath } from 'path'
-import { Admin } from 'web3-eth-admin'
 import {
   AccountType,
-  generateGenesis,
   getPrivateKeysFor,
-  getValidators,
-  Validator,
+  getValidatorsInformation,
+  privateKeyToAddress,
+  privateKeyToPublicKey,
 } from '../lib/generate_utils'
-import { ensure0x } from '../lib/utils'
-
-export interface GethInstanceConfig {
-  name: string
-  validating: boolean
-  syncmode: string
-  port: number
-  rpcport?: number
-  wsport?: number
-  lightserv?: boolean
-  privateKey?: string
-  etherbase?: string
-  peers?: number[]
-  pid?: number
-}
-
-export interface GethTestConfig {
-  migrate?: boolean
-  migrateTo?: number
-  instances: GethInstanceConfig[]
-  genesisConfig?: any
-  migrationOverrides?: any
-}
+import {
+  buildGeth,
+  checkoutGethRepo,
+  connectValidatorPeers,
+  getEnodeAddress,
+  GethInstanceConfig,
+  GethRunConfig,
+  initAndStartGeth,
+  resetDataDir,
+  restoreDatadir,
+  snapshotDatadir,
+  spawnWithLog,
+  startGeth,
+  writeGenesis,
+} from '../lib/geth'
+import { ensure0x, spawnCmd, spawnCmdWithExitOnFailure } from '../lib/utils'
 
-const TEST_DIR = '/tmp/e2e'
-const GENESIS_PATH = `${TEST_DIR}/genesis.json`
-const NetworkId = 1101
```

Comment:
Why were these extracted from this file? I didn't see them changes anywhere? It seems reasonable to specify the defaults only in this file.